src/terminal/parser/stateMachine.cpp

```diff
@@ -86,24 +86,14 @@ static constexpr bool _isC0Code(const wchar_t wch) noexcept
 // - wch - Character to check.
 // Return Value:
 // - True if it is. False if it isn't.
-static constexpr bool _isC1Csi(const wchar_t wch) noexcept
+static constexpr bool _isC1ControlCharacter(const wchar_t wch) noexcept
 {
-    return wch == L'\x9b';
+    return (wch >= L'\x80' && wch <= L'\x8F') || (wch >= L'\x90' && wch <= L'\x9F');
 }
 
-// Routine Description:
-// - Determines if a character is a C1 DCS (Device Control Strings)
-//   This is a single-character way to start a control sequence, as opposed to "ESC P".
-//
-//   See the comment above _isC1Csi for more information on how this is impacted by codepages.
-//
-// Arguments:
-// - wch - Character to check.
-// Return Value:
-// - True if it is. False if it isn't.
-static constexpr bool _isC1Dcs(const wchar_t wch) noexcept
+static constexpr wchar_t _c1To7Bit(const wchar_t wch) noexcept
 {
-    return wch == L'\x90';
+    return wch - L'\x40';

```

Comment:
This *seems* correct but I can't find valid reference for this.

Reply:
Yeah, I believe this is correct. Quoting the `DEC ST 070` manual:
> expansion escape sequence (Fe) - 2-character escape sequence in which the final character is in columns 4 and 5, and is used to extend the 7-bit code table by being the row equivalent to the corresponding C1 set of controls in columns 8 and 9 of an 8-bit code.

Reply:
(Should we prefer `-0x40` or `&~0x40` here?)

Reply:
I choose `L'\x40'` cuz I want it have the same type as `wch`. Not that it mean anything, but hey, no implicit conversion.